posts/2022-02-15-Rust-Survey-2021.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "Rust Survey 2021 Results"
+author: The Rust Survey Team
+---
+
+# Blog Post
+
+Greetings Rustaceans!
+
+Another year has passed, and with it comes another annual Rust survey analysis! The survey was conducted in December 2021.
+
+We’d like to thank everyone who participated in this year’s survey with a special shout-out to those who helped translate the survey from English into other languages.
+
+Without further ado, let’s dive into the analysis!
+
+## A Global Community
+
+The Rust community continues to grow, with this survey having the largest number of complete survey responses (9354 respondents), exceeding last year's total by roughly 1500 responses.
+
+90% of respondents said that they use Rust for any purpose, while 5% stated they had used Rust at some point in the past but no longer do, and 4% stated they have yet to use Rust at all.
+
+The survey was offered in 10 languages with 78% filling out the survey in English followed by Chinese (6%), German (4%), and French (3%). Despite English being the language most respondents completed the survey in, respondents hailed from all around the world. The United States was the country with the largest representation at 24% followed by Germany (12%), China (7%), and the U.K. (6%). In total 113 different countries and territories were represented through this survey!
+
+English, however, is not the language of choice for all Rustaceans with nearly 7% preferring not to use English as a language for technical communication. An additional 23% of respondents prefer another language in addition to English. The most commonly preferred languages (besides English) roughly follow where Rustaceans live with Chinese, German, and French being the top 3. However, Japanese, Chinese, and Russian speakers were the most likely to prefer not to use English at all for technical conversation.
+
+## Using Rust
+
+The percentage of people using Rust continues to rise. Of those using Rust, 81% are currently using it on at least a weekly basis compared to 72% from last year's survey.
+
+75% of all Rust users say they can write production ready code, though 27% say that it is at times a struggle.
+
+![Rust expertise](/images/2022-02-RustSurvey/expertise.png)
+
+Overall, Rustaceans seem to be having a great time writing Rust with only 1% saying it isn't fun to use. Only a quarter of a percent find Rust doesn't have any real benefit over other programming languages.
+
+## Rust at Work
+
+Rust can now safely be classified as a language used by people in professional settings. 59% of respondents who currently use Rust, use it at least occasionally at work with 23% using Rust for the majority of their coding. This is a large increase over last year where 58% of respondents didn't use Rust at work compared with this year's 41%.
```

Comment:
I had to read this paragraph several times. It particularly confused me that it switched from "percent that do use rust" to "percent that don't" when doing a comparison. Suggested rewrite.

```suggestion
Rust can now safely be classified as a language used by people in professional settings. Of those respondents using Rust, 59% use it at least occasionally at work with 23% using Rust for the majority of their coding. This is a large increase over last year where only 42% of respondents used Rust at work.
```